specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/examples/WafListAllPolicies.json

```diff
@@ -148,6 +148,69 @@
                     "ruleSetType": "OWASP",
                     "ruleSetVersion": "3.2"
                   }
+                ],
+                "allowlists": [
+                  {
+                    "matchVariable": "RequestURI",
+                    "valueMatchOperator": "Contains",
+                    "values": [
+                      "path1",
```

Comment:
In your examples, you have used illustrative IPs when IP is to be specified. For end users, it will be better to use real URIs

Reply:
Updated to use real URIs.
Note: AppGW WAF works with the URI Path and query string. It doesn't operate on the domain.